### ISSUE TITLE
Autoresize list of suggestions

### ISSUE THREAD

**Issue by VivienGiraud** [EurekaCommunity/SuggestionRow#5]
Hi,

Just change:

>tableView?.frame = CGRect(x: 0, y: frame.origin.y + frame.height, width: contentView.frame.width, height: 44*5)

To

>tableView?.frame = CGRect(x: 0, y: frame.origin.y + frame.height, width: contentView.frame.width, height: 44 * CGFloat((suggestions?.count)!))

In function `showTableView()`  
A bit quick&dirty but that works.

**Comment by mats-claassen** [comment]
Yeah that `5` is not very nice. A possible solution would be to add a new variable to `SuggestionRow` which defines how many rows will be shown in the suggestion table as maybe `suggestions.count` is too large.

**Comment by VivienGiraud** [comment]
TBH `suggestions.count` works great, I'm just looking to replace that `44` :)

**Comment by VivienGiraud** [comment]
I understand what you meant by "is too large", then you can do the "Quick & Dirty AF" version:

> let frame = controller.tableView?.convert(self.frame, to: controller.view) ?? self.frame
            tableView?.frame = CGRect(x: 0, y: frame.origin.y + frame.height, width: contentView.frame.width, height: 44 * (((suggestions?.count)! <= 5) ? CGFloat((suggestions?.count)!) : 5))

**Comment by mats-claassen** [comment]
Well you are correct in that the 44 should also be parametrizable.
I believe this problem is best fixed if we add two variables with sensible default values instead of using hardcoded 44 and 5. The variable for the amount of rows would be a maximum like so:
`height = min(suggestions.count, maxNumberOfRows)`

What do you think?

The question that arises then is what if there are no suggestions? Show x empty rows? Showing nothing might not look good.

A possibility  would be:
`height = max(3, min(suggestions.count, maxNumberOfRows))`
